### ISSUE TITLE
Dark theme how to set.

### ISSUE THREAD

**Issue by djmcg** [arduino/arduino-pro-ide#403]
Hello
I`m try to change default theme for dark.
Could You advice how to do it.?>

**Comment by djmcg** [comment]
Finally I find it:
`Ctrl+Shift+P
![Snap3](https://user-images.githubusercontent.com/27962351/102711206-1776a800-42b8-11eb-92ba-a4cb9b2687e3.png)
`

**Comment by ubidefeo** [comment]
hi @djmcg 
the theme setting belongs to Theia and being still in alpha we don't have many functionalities exposed yet.
happy you found it, consider this an Easter Egg... or a 🎄 one? 😬

**Comment by NoSmoke999** [comment]
Ctrl+Shift+P gives a pop-up for all the keyboard shortcuts on my version (0.1.3).  Has there been a new release?

**Comment by ubidefeo** [comment]
@NoSmoke999 
correct, with that shortcut you get access to a lot of Theia configuration stuff.
When I say "exposed" I mean through a more user-friendly UI
0.1.3 is the latest

**Comment by sebasdt** [comment]
> Ctrl+Shift+P gives a pop-up for all the keyboard shortcuts on my version (0.1.3). Has there been a new release?

look in preferences or CTRL+Komma. and theme

**Comment by ubidefeo** [comment]
@sebasdt the Nightly Build has Themes selectable from the Preferences GUI ;)
